test.py

```diff
@@ -0,0 +1,28 @@
+### for testing modules that require other modules in different folders ###
```

Comment:
this should go in the `tests` folder and have a more specific name ;)

Reply:
I believe this was added 'accidentally' via git add .

Please refer to my previous comment regarding a possible testing widget. In the interim it has been move to tests folder